system/draft/test_misc.py

```diff
@@ -96,28 +90,67 @@
     print(res)
 
 
+# TODO parametrize this to read as trustee and as default user
 @pytest.mark.asyncio
 async def test_misc_state_proof(
```

Comment:
What exactly does this test checks? Please consider writing a short description for every test.
For example:
- add X NYM txns
- read written NYM txns
- make sure that ....

Reply:
It will be in the next PR. Also it will be a separate test.